spec/models/user/email_spec.rb

```diff
@@ -42,5 +42,16 @@
         expect(subject.errors[:email].count).to eq(1)
       end
     end
+
+    context 'when the email is not confirmed' do
+      subject { create(:user_email, :unconfirmed) }
+
+      it 'sends email with ActiveJob queue' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery)
```

Comment:
I see, that's because `expect(Devise::Mailer).to receive(:confirmation_instructions).and_return(message_delivery)` is needed, that line stubs the `Devise::Mailer` and let it returns the `instance_double`.